X/XicTools/build_tarballs.jl

```diff
@@ -0,0 +1,130 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "XicTools"
+version = v"0.1.0"
```

Comment:
There is a strong reason in this case because builds without a GUI (such as gtk2) were only introduced in the latest commit as discussed in this issue: https://github.com/wrcad/xictools/issues/18 . I plan to shift to the latest release as soon as they tag one.

Reply:
I see, but it'd be nicer to have a version number "close" to upstream, like 4.3.18 or 4.3.19?

Reply:
Sure, no problem. I'll change to 4.3.19.